appveyor.yml

```diff
@@ -30,6 +25,14 @@ environment:
       CONDA_PY: 35
       CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
 
+    - TARGET_ARCH: x86
+      CONDA_PY: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda36
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
```

Comment:
Not sure why this is here as the `skip` should have ensure it wasn't added. Could you please try another re-render to see if that removes this? If not, we will need to investigate.